lib/scala/pkg/src/main/scala/org/enso/pkg/Package.scala

```diff
@@ -265,7 +263,7 @@ class PackageManager[F](implicit val fileSystem: FileSystem[F]) {
     template: Template
   ): Package[F] = {
     val pkg = new Package(root, config, fileSystem)
-    pkg.save()
+    pkg.save().get
```

Comment:
Discarding the result of `Try[...]`... - not the best idea

Reply:
What's the point of using `Try` there, then? `.get` will crash with an exception on failure and not really report the failure either.